projects/aca-content/viewer/src/lib/components/viewer/viewer.component.ts

```diff
@@ -74,62 +75,37 @@ export class AcaViewerComponent implements OnInit, OnDestroy {
 
   fileName: string;
   folderId: string = null;
-  nodeId: string = null;
-  versionId: string = null;
-  node: Node;
-  selection: SelectionState;
   infoDrawerOpened$: Observable<boolean>;
-
-  showRightSide = false;
-  openWith: ContentActionRef[] = [];
-  toolbarActions: ContentActionRef[] = [];
-
+  navigateMultiple = true;
   navigateSource: string = null;
-  previousNodeId: string;
+  navigationSources = ['favorites', 'libraries', 'personal-files', 'recent-files', 'shared'];
   nextNodeId: string;
-  navigateMultiple = true;
+  node: Node;
+  nodeId: string = null;
+  openWith: ContentActionRef[] = [];
+  previousNodeId: string;
   routesSkipNavigation = ['shared', 'recent-files', 'favorites'];
```

Comment:
As you are doing changes here, could you remove that field because it seems unused?